R/pkg/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: SparkR
 Type: Package
-Version: 3.0.0
+Version: 3.0.0.p0
```

Comment:
This seems to cause a failure.
```
* installing *source* package ���SparkR��� ...
Error : Invalid DESCRIPTION file

Malformed package version.

See the information on DESCRIPTION files in section 'Creating R
packages' of the 'Writing R Extensions' manual.

ERROR: installing package DESCRIPTION failed for package ���SparkR���
```

Reply:
It seems `3.0.0.p0 / 3.0.0.p / 3.0.0.0p` are not malformed package versions. cc @HyukjinKwon